### PR TITLE
fix example id to match file name

### PR DESCRIPTION
**Pull request by ryma2fhir**
To ensure no duplicate/incorrect ids the naming system checker now checks that example ids match the file name

### CODE REVIEW

File: examples/UKCore-CareTeam-WeightManagementTeamExample.xml
Comment (APageNHS) [inline examples/UKCore-CareTeam-WeightManagementTeamExample.xml:1]: Probably better to rename the file on this one

File: examples/UKCore-ServiceRequest-Extension-PriorityReason_SendingAsText-Example.xml
Comment (APageNHS) [inline examples/UKCore-ServiceRequest-Extension-PriorityReason_SendingAsText-Example.xml:1]: Should we change the filename on this one?

Comment (ryma2fhir) [reply examples/UKCore-ServiceRequest-Extension-PriorityReason_SendingAsText-Example.xml:1]: It makes sense as we don't use underscores anywhere else. I'm thinking another hypen.

Comment (APageNHS) [reply examples/UKCore-ServiceRequest-Extension-PriorityReason_SendingAsText-Example.xml:1]: I'd be ok with the file being PriorityReasonString

File: examples/UKCore-MedicationDispense-CiprofloxacinOral-Example.xml
Comment (APageNHS) [inline examples/UKCore-MedicationDispense-CiprofloxacinOral-Example.xml:1]: can you also update display to "Ciprofloxacin 500mg tablets (Accord-UK Ltd)"

**Comment by ryma2fhir** [comment]
Probably should have done it on another branch but I've removed the profile name from all extension examples, so for example:
UKCore-Patient-Extension-NHSNumberUnavailableReason-Example
has become
UKCore-Extension-NHSNumberUnavailableReason-Example

This follows the convention of the extension asset itself (eg. UKCore-Extension-NHSNumberUnavailableReason) and helps with the character limit error we've had on numerous occasions.

**Review by APageNHS — APPROVED** [review]

**Pull request merged**